src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferencesOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.FindSymbols
+{
+    internal class FindReferencesOptions
```

Comment:
💡 Consider making this either `sealed` or `readonly struct`

Reply:
👍

Reply:
ack... this type shouldn't be here.